src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -99,7 +99,8 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
   get startingAddress() {
     return {
       ...emptyAddress,
-      country: "US",
+      country: this.props.order.lineItems.edges[0].node.artwork.shippingCountry,
```

Comment:
As long as we're 100% sure this will always be non-null 👍, otherwise it might be good to add some save-access operators across this long chain.

Reply:
@damassi that's a good question, and I'd appreciate your 🧠   here. 

A little lower down in the same file in the `render()` call, we have [these lines](https://github.com/artsy/force/blob/8cca9fab9f695ba680627a0dfb7639580e64eae6/src/v2/Apps/Order/Routes/Shipping/index.tsx#L337-L340):

```js
const artwork = get(
  this.props,
  props => props.order.lineItems.edges[0].node.artwork
)
```

I'm a little confused by this - we're using [get()](https://github.com/artsy/force/blob/8cca9fab9f695ba680627a0dfb7639580e64eae6/src/v2/Utils/get.ts), which is meant to provide "safe access" by attempting to get a property and returning a `valueIfFailOrUndefined` if that fails, which is an optional third parameter. But...we're not passing a third parameter here. I tested this on my own and if we try to `get` something invalid without passing a `valueIfFailOrUndefined`, we just end up with `undefined`.

**So TL;DR:** it seems like we're already assuming `props.order.lineItems.edges[0].node.artwork` will always exist. 

To be fair, that makes sense - we're entering the checkout flow for an artwork, so if we somehow don't have an artwork, we're in trouble.

To your question about `shippingCountry`: I'm confident that this exists for BNMO works because we require a shipping address in CMS and `shippingCountry` is basically [just that](https://github.com/artsy/metaphysics/blob/d33a34b11dba69edd87336934e39165edb40c9a1/src/schema/v2/artwork/index.ts#L684-L693)

Reply:
Sounds good 👍 

And regarding `get`, its mostly used as a guard so things don't blow up and I've personally never noticed anyone use the fallback arg.